Vr interface update

garraww opened the pull request:
- Ajout des menus
- Ajout base mobile
- Contrôle base temps réel/manuel
- Ajout publisher/Subscriber

garraww committed:
Initial Commit

garraww committed:
Initial commit

garraww committed:
Initial Commit Fix

garraww committed:
Ajout des boutons manuel pour la base mobile

will-44 commented:
Pourquoi il y a deux dossiers avec le meme nom + update ?

garraww commented:
Où exactement tu vois deux dossiers avec le même nom ? Je ne le retrouve pas ?

will-44 commented:
<img width="289" height="249" alt="Screenshot from 2025-07-11 13-38-02" src="https://github.com/user-attachments/assets/232f5918-d915-48b6-8d8e-11ff8d3a1a7f" />

will-44 commented:
En fait, pourquoi avoir renommé le dossier en VR_Zorg_Etude_update. Il faut garder le même nom sauf s'il y a une réelle raison

garraww commented:
En fait c'était pour différencier de notre côté, l'original de celui sur lequel on a travaillé. Mais je vais modifier tous ça aujourd'hui.

garraww commented:
> En fait, pourquoi avoir renommé le dossier en VR_Zorg_Etude_update. Il faut garder le même nom sauf s'il y a une réelle raison

Si je modifie le nom du dossier tout les fichiers sont affectés et il y'aura plus de 10000 changements dû à des changements de path. Est ce que je modifie tout de même le nom du dossier ?

will-44 commented:
Garde le nom de base

garraww committed:
Add the original folder name

will-44 commented:
Est-ce que tu peux me faire une liste des fichiers/dossiers que tu as modifiés ? (ça m'évitera de vérifier les 1000 nouveaux fichiers)

will-44 commented:
Pourquoi le readme a été supprimé et est-ce que vous pouvez le mettre à jour en fonction des nouvelles fonctionnalités.

will-44 commented:
Il faut commenter les fichiers pour expliquer le fonctionnement. Prenez exemple sur les commentaires du projet initial

garraww commented:
> Est-ce que tu peux me faire une liste des fichiers/dossiers que tu as modifiés ? (ça m'évitera de vérifier les 1000 nouveaux fichiers)

<img width="603" height="340" alt="image" src="https://github.com/user-attachments/assets/9c266895-09c8-4d4e-9992-6fd320ae38ef" />

<img width="604" height="58" alt="image" src="https://github.com/user-attachments/assets/3abe31d3-a496-427f-b5aa-b63e72f6f770" />

C'est les grosses modifications qu'on a fait.

will-44 commented on VR_Zorg_Etude/Assets/URDF/New_version_Doosan_m1013.meta:
Pourquoi ce fichier s'appelle New_vesion et pas juste doosan_m1013 ?

garraww replied:
Je pense que cela concerne un URDF du site web Doosan  que nous avions ajouté pour vérifier si les liens entre joints étaient à jour. Il n'est pas utilisé, il fait simplement partie des assets disponibles.

will-44 commented on VR_Zorg_Etude/Assets/URDF/New_version_Doosan_m1013/dsr_description.launch.py:
Pourquoi il y a un launch file ici ?

garraww replied:
Pareil, cela concerne une tentative d'ajout de l'URDF du robot (mis-à-jour) pour vérifier si le problème des joints du robot, qui étaient inversés, pouvait être résolu. Finalement le problème ne venait pas de là.

garraww committed:
Amelioration documentation code

will-44 commented on README.md:
Ce fichier est à mettre à jour suivant vos modifications. Surtout, comment exécuter le projet ?

will-44 commented on VR_Zorg_Etude/Assets/Scripts/ROS/RosPubMRTwist.cs:
Quel est la difference entre RosPubMRTwist et RosPubMRTwistSingle ?

garraww replied:
RosPubMRTwist permet le controle en temps réeel (avec manette) de la base mobile.
RosPubMRTwistSingle permet le controle manuel (à partir du menu) de la base mobile.

will-44 commented on VR_Zorg_Etude/Assets/Scripts/ROS/RosPubMRTwist_Single.cs:
```diff
@@ -3,31 +3,39 @@
     private bool y_transOnly = false;
     private bool rotOnly = false;
 
     private int moveDirection = 0; // 1 = avancer, -1 = reculer, 0 = arrêt
```
Faire une enum pour plus de lisibilité

garraww committed:
Correction des bugs pour la base mobile

- Translation fluide base mobile :
RosSubMRTwist.cs

- Faire rotation en fonction du cube :
RosSubMRTwist.cs

- Changer bouton vitesse de rotation :
PlusMinusButton
ManualSpeedController
RosPubMRTwist_Single

Todo :
- nettoyer code, Corriger nom variable, ajouter commentaire
- Modifier Read Me
- corriger le mode en temps réelle

garraww committed:
Correction Bouton de vitesse

garraww committed:
Correction des bugs et nettoyer le code pour la base mobile

Correction des bugs pour la base mobile
- Temps réelle et Manuel ne s'intercepte plus
- Correction des vitesses
- Correction des axes

Nettoyage du bugs
- RosPubMRTwist.cs
- RosPubMRTwist_Single.cs
- RosSubMRTwist.cs

garraww committed:
Modification Vitessse temps réel.

- Placement correcte des bouton de vitesse (ui)
- Ajout bouton vitess temps réel
- Meilleur fluidité déplacement

garraww committed:
Ajout message Srv MoveBlending, modification scale robot
